plugins/wpe-headless/includes/rest/callbacks.php

```diff
@@ -18,24 +18,26 @@
  *
  * @link https://www.wpgraphql.com/filters/graphql_authentication_errors/
  *
+ * @param bool $authentication_errors Whether there are authentication errors with the request.
```

Comment:
This whole file got merged into `main` [here](https://github.com/wpengine/headless-framework/commit/eb40fb52ee82524bf24f63a8059d98bd8acea47a), so not sure why it's showing. Not a problem, just curious. LGTM.

Reply:
I think it's the difference between pulling in all of `canary` vs a specific branch like `BH-722` in https://github.com/wpengine/headless-framework/pull/42